docs/UserGuide.md

```diff
@@ -190,9 +186,11 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [g/GENDER] [age/AGE
 </div>
 
 Examples:
-* `edit 1 n/Malan i/[2] Swimming` edits the name of contact and the second interest of the first contact in the displayed list
-* `edit 1 i/Painting i/[1] Running` adds 'painting' as an interest and edits the first interest of the first contact in the displayed list
+* `edit 1 n/Malan i/(2) Swimming` edits the name of contact and the second interest of the first contact in the displayed list
+* `edit 1 i/Painting i/(1) Running` adds 'painting' as an interest and edits the first interest of the first contact in the displayed list
 * `edit 2 g/M e/myEmail@email.com age/55` edits the gender, email and age of the second contact in the displayed list
+* `edit 3 i/golf i/(1) eat i/(2) remove` removes the second interest in the interests list, edits the first item of the interests list to 'eat' 
```

Comment:
"edits the first item...." should be "edits the first interest"

Reply:
Dont have to say "interests list" everytime. Just the first time will do. for eg. the second one can just be "edits the first interest to 'eat'"